### DISCUSSION TITLE
ani-cli works in repository but not directly through terminal

### DISCUSSION THREAD

**Discussion by aj-arts** [pystardust/ani-cli#271] [Q&A] [answered]
I am on intel Mac and I use ani-cli a lot. it stopped working so I push origin in GitHub desktop and opened the new ani-cli file in the repository it works fine but when I type ani-cli when I am not in my repository it doesn't work.

**Comment by Dink4n** [comment]
Did you run `sudo make install`?

**Reply by aj-arts** [reply]
it worked I manually copy pasted the file to local/bin and now it works😌😌

**Comment by port19x** [comment, marked as answer]
and that is why I want to get rid of the makefile
